main.tf

```diff
@@ -114,6 +113,10 @@ resource "aws_cloudwatch_log_group" "this" {
     var.tags,
     { Name = "/aws/eks/${var.cluster_name}/cluster" }
   )
+
+  depends_on = [
```

Comment:
this will fail everytime - the cluster will be created, and it will create the log group and Terraform will fail with a "resource with the same name already exists"

Reply:
Good point... didn't think of that... there has to be a way to fix this issue some other way since the deny doesn't appear to be working as expected. I know it's implemented different than the original PR, curious as to if that's making a difference in some way.